hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1275,7 +1275,7 @@ private void startTrashEmptier(Configuration conf) throws IOException {
 
     // configuration for the FS instance that  points to a root OFS uri.
     // This will ensure that it will cover all volumes and buckets
-    Configuration fsconf = new Configuration();
+    Configuration fsconf = new OzoneConfiguration();
```

Comment:
I think this is intentional. @elek @bharatviswa504 can you please confirm this line ?

Reply:
From my understanding, we need to use OzoneConfiguration, as OzoneConfiguration activates ozone-default.xml

  public static void activate() {
    // adds the default resources
    Configuration.addDefaultResource("hdfs-default.xml");
    Configuration.addDefaultResource("hdfs-site.xml");
    Configuration.addDefaultResource("ozone-default.xml");
  }

By using Configuration, defaults of Ozone will not be loaded. And also as we have for each config, we have defaults defined in Java code that should also be fine.

Using OzoneConfiguration is better IMHO.

Reply:
thx @bharatviswa504